rest/context.go

```diff
@@ -192,44 +191,56 @@ func NewContext(parent context.Context, req *http.Request, writer http.ResponseW
 	// parameters with the same name as query string values. Figure out a
 	// better way to handle this.
 
-	return &gorillaRequestContext{parent, req, bytes.NewBuffer(body), writer, nil, []string{}}
+	return &requestContext{
+		req:      req,
```

Comment:
do you need this?
```go
Context:  req.Context(),
```

Reply:
🤦 Removing to force explicit use of `req.Context()`